packages/lane_filter/include/lane_filter/lane_filter.py

```diff
@@ -134,7 +131,7 @@ def predict(self, dt, v, w):
         d_t = self.d + v * delta_t * np.sin(self.phi)
         phi_t = self.phi + w * delta_t
 
-        for k in range(self.curvature_res):
+        for k in range(self.curvature_res + 1):
```

Comment:
This is the only change, the rest is autoformat.
Since curvature res is always 0, this loop was never called, meaning that the predict step did nothing